packages/turf-square-grid/index.ts

```diff
@@ -29,56 +21,19 @@ import {getType} from "@turf/invariant";
  * //addToMap
  * var addToMap = [squareGrid]
  */
-function squareGrid<P = Properties>(bbox: BBox, cellSide: number, options: {
-    units?: Units,
-    properties?: P,
-    mask?: Feature<Polygon | MultiPolygon> | Polygon | MultiPolygon,
-} = {}): FeatureCollection<Polygon, P> {
-    // Containers
-    const results = [];
-    const west = bbox[0];
-    const south = bbox[1];
-    const east = bbox[2];
-    const north = bbox[3];
 
-    const xFraction = cellSide / (distance([west, south], [east, south], options));
-    const cellWidth = xFraction * (east - west);
-    const yFraction = cellSide / (distance([west, south], [west, north], options));
-    const cellHeight = yFraction * (north - south);
-
-    // rows & columns
-    const bboxWidth = (east - west);
-    const bboxHeight = (north - south);
-    const columns = Math.floor(bboxWidth / cellWidth);
-    const rows = Math.floor(bboxHeight / cellHeight);
+import {
```

Comment:
this should go at the top, same as above, just for consistency within the library